BuildAndTest.cmd

```diff
@@ -5,15 +5,25 @@ SETLOCAL
 @REM create a nuget package for binskim) so must opt-in
 @REM %~dp0.nuget\NuGet.exe update -self
 
+set Platform=Any CPU
+set Configuration=Release
+
+:NextArg
+if "%1" == "" goto :EndArgs
+if "%1" == "/config" (
+    if not "%2" == "Debug" if not "%2" == "Release" echo error: /config must be either Debug or Release && goto :ExitFailed
+    set Configuration=%2&& shift && shift && goto :NextArg
```

Comment:
The lack of white space after "`%2`" is intentional. Any white space becomes part of the value of `Configuration`.